lib/rake/application.rb

```diff
@@ -165,7 +165,13 @@ def add_loader(ext, loader)
 
     # Application options from the command line
     def options
-      @options ||= OpenStruct.new
+      @options ||= Struct.new(
+        :always_multitask, :backtrace, :build_all, :dryrun,
+        :ignore_deprecate, :ignore_system, :job_stats, :load_system,
+        :nosearch, :rakelib, :show_all_tasks, :show_prereqs,
+        :show_task_pattern, :show_tasks, :silent, :suppress_backtrace_pattern,
+        :thread_pool_size, :trace, :trace_output, :trace_rules
+      ).new
```

Comment:
It would be nicer if this was a named class:

```
NoMethodError: undefined method `verbose' for an instance of #<Class:0x00000001038112e0>
```

vs something like:

```
NoMethodError: undefined method `verbose' for an instance of #<Rake::Options:0x00000001038112e0>
```